lib/infer/membership.js

```diff
@@ -77,6 +116,32 @@ function countModuleIdentifiers(comment, identifiers) {
 }
 
 /**
+ * Returns the comment object after normalizing Foo.prototype and Foo# expressions
+ * @param {Object} comment parsed comment
+ * @returns {Object} the normalized comment
+ */
+function normalizeMemberof(comment) {
+  var memberof = comment.memberof;
+  var index = memberof.lastIndexOf('.');
+
+  if (index !== -1 && memberof.slice(index + 1) === 'prototype') {
```

Comment:
could these index-based checks be expressed more simply with regular expressions? I believe this one would be `memberof.match(/\.prototype$/)`